src/hotspot/share/gc/parallel/spaceCounters.cpp

```diff
@@ -59,8 +60,9 @@ SpaceCounters::SpaceCounters(const char* name, int ordinal, size_t max_size,
                                    _object_space->capacity_in_bytes(), CHECK);
 
     cname = PerfDataManager::counter_name(_name_space, "used");
+    _perf_used_helper = new MutableSpaceUsedHelper(_object_space);
     _used = PerfDataManager::create_variable(SUN_GC, cname, PerfData::U_Bytes,
-                                    new MutableSpaceUsedHelper(_object_space),
+                                    _perf_used_helper,
```

Comment:
Pre-existing: the formatting of `PerfDataManager::create*` calls is rather unusual. I filed [JDK-8310116](https://bugs.openjdk.org/browse/JDK-8310116) for that.